tests/end_to_end/test_vulnerability_detector/test_vulnerability_detector.py

```diff
@@ -109,6 +109,70 @@ def load_vulnerability_detector_configurations(host_manager):
 
     return configurations
 
+@pytest.fixture(scope='module')
+def install_npm(host_manager: HostManager):
+    """Ensure npm is installed on macOS agents."""
```

Comment:
This fixture does not ensure npm is installed, it install it

Reply:
Fixed here: https://github.com/wazuh/wazuh-qa/pull/5158/commits/998c839e00a48bccc54fa00cb95cf87b30616b0a